TestProjects/com.unity.mobile-notifications-sample/Scripts/ScriptableObjects/iOSNotificationTemplateTimeTrigger.cs

```diff
@@ -36,6 +36,26 @@ public class iOSNotificationTemplateTimeTrigger : ScriptableObject
         [Header("Time Trigger")]
         public Int32 TimeTriggerInterval;
         public bool Repeats = false;
+
+        [Serializable]
+        public struct UserDataItem
+        {
+            public string Key;
+            public string Value;
+        }
+        public UserDataItem[] UserInfo;
```

Comment:
This variable doesn't look like it supposed be accessed, since it looks like you're indenting for user to access UserInfoDict? 

If so, maybe
[SerializeField]
private UserDataItem[] m_UserInfo;

and rename UserInfoDict to UserInfo.

Please ignore if I am wrong

Reply:
This is sample-only file. This field needs to be either public or serializable so it can be edited in inspector.